src/components/TaskList.tsx

```diff
@@ -16,25 +21,23 @@ export function TaskList() {
   }
 
   return (
-    <>
-      {/* Start: this should be deleted whenever you're comfortable */}
-      <Stack spacing={4} justifyContent="center" alignItems="center" h="full">
-        <Text color="white">UCU x Fluxon Product Development Bootcamp</Text>
-        <HStack mt={4} color="blue.100">
-          <Link href="/firebase-demo">Firebase demo</Link>
-          <Text>|</Text>
-          <Link href="/auth">Authenticate</Link>
-          <Text>|</Text>
-          <Link href="/tasks">Tasks</Link>
-        </HStack>
-      </Stack>
-      {/* End: this should be deleted whenever you're comfortable */}
-
+    <MainLayout
+      headerContent={
+        <>
+          <Button onClick={signOut} isDisabled={isSigningOut} isLoading={isSigningOut}>
+            Sign out
+          </Button>
+          <Button as={Link} href="/tasks" ml={2}>
+            Tasks
+          </Button>
```

Comment:
I think this is not needed as we will have only 1 page with a header and it will be a page with the tasks list. So we don't need a `/tasks` menu item in a header.

That makes the header to look the same and contain only the `Sign out` button for now. It means that you can render the `<Button ...>Sign out </Button>` inside of the `MainLayout` component and remove the `headerContent` at all.